lib/cfnguardian/resources/dms_task.rb

```diff
@@ -0,0 +1,42 @@
+module CfnGuardian::Resource
+  class DMSTask < Base
+    
+    def default_alarms             
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencySourceCritical'
+      alarm.metric_name = 'CDCLatencySource'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 10
+      alarm.enabled = false
+      @alarms.push(alarm)
+      
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencySourceWarn'
+      alarm.metric_name = 'CDCLatencySource'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 1
+      alarm.enabled = false
+      @alarms.push(alarm)
+
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencyTargetCritical'
+      alarm.metric_name = 'CDCLatencyTarget'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 10
+      alarm.enabled = false
+      @alarms.push(alarm)
+
+      alarm = CfnGuardian::Models::DMSTaskAlarm.new(@resource)
+      alarm.name = 'CDCLatencyTargetWarn'
+      alarm.metric_name = 'CDCLatencyTarget'
+      alarm.statistic = 'Minimum'
+      alarm.threshold = 30
+      alarm.evaluation_periods = 1
```

Comment:
is 1 threshold to low?

Reply:
Jared and I agreed on 1 so that _if_ the latency is falling behind 30 seconds, even just briefly, ideally we'd want to know about it so we can see if it's happening regularly